src/FirstRun/LanguagePickerScreen/LanguagePickerScreen.tsx

```diff
@@ -9,17 +9,17 @@ import {FirstRunRouteNavigation} from '../../navigation'
 
 export const LanguagePickerScreen = () => {
   const navigation = useNavigation<FirstRunRouteNavigation>()
-  const intl = useIntl()
+  const strings = useStrings()
 
   return (
     <SafeAreaView style={styles.safeAreaView}>
```

Comment:
If it's inside the navigator, you have to drop `top` on edges, otherwise it will have an extra space reserved to the navigation bar+status bar on top. The new version of the navigator also controls when using the bottom bar.
```suggestion
    <SafeAreaView edges={['left', 'right', 'bottom']} style={styles.safeAreaView}>
```